.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
         include:
           # Test MSRV
-          - rust: 1.40.0
+          - rust: 1.51.0
```

Comment:
For me requiring 1.51 at this point in time is not ideal. Other opinions, @adamgreig maybe?

Reply:
Is this to generate crates or to use the generated crates? 1.51 is _way_ too new for building PACs, but I wouldn't mind requiring it to generate them.

Reply:
Generated crates. It's providing a const generic writer. I would not terribly object if this was hidden behind a feature gate but I fully agree that enabled for all is a non-starter.

Reply:
> behind a feature gate

It's idea

Reply:
Done

Reply:
Now this PR doesn't change anything if you don't use `const-generic` feature.